### PR TITLE
Added Evo 15m and 40m

### PR DESCRIPTION
**Pull request by mapi-ng**

### CODE REVIEW

**Comment by pptaszni** [comment]
Hey guys, pay attention when copy-pasting code from other sensors, especially author name, copyright year, and sensor names in the outputs :)

File: examples/ExampleReadEvo15m.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+volatile std::sig_atomic_t g_signal_status;
+void signal_handler(int signal) {
+  g_signal_status = signal;
+}
+
+int main(int argc, char** argv)
+{
+  if (argc != 2)
+  {
+    std::cout << "usage: ./ExampleReadEvo60 DEVICE_NAME" << std::endl;
```
Comment (pl-kabaradjian) [inline examples/ExampleReadEvo15m.cpp]: Fix sensor name

File: examples/ExampleReadEvo40m.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+volatile std::sig_atomic_t g_signal_status;
+void signal_handler(int signal) {
+  g_signal_status = signal;
+}
+
+int main(int argc, char** argv)
+{
+  if (argc != 2)
+  {
+    std::cout << "usage: ./ExampleReadEvo60 DEVICE_NAME" << std::endl;
```
Comment (pl-kabaradjian) [inline examples/ExampleReadEvo40m.cpp]: Fix sensor name

File: include/terabee/internal/teraranger/TerarangerEvo15m.hpp [hunk 3]
```diff
@@ -0,0 +1,47 @@
+#ifndef TERARANGER_EVO_15M_HPP
+#define TERARANGER_EVO_15M_HPP
+
+/**
+ * @Author Pawel Ptasznik
+ *
+ * @Copyright Terabee 2020
+ *
+ */
```
Comment (pl-kabaradjian) [inline include/terabee/internal/teraranger/TerarangerEvo15m.hpp:9]: Fix author and copyright year

File: include/terabee/internal/teraranger/TerarangerBasicCommon.hpp [hunk 4]
```diff
@@ -6,8 +6,12 @@
  *
  * @Copyright Terabee 2020
```
Comment (pl-kabaradjian) [inline include/terabee/internal/teraranger/TerarangerBasicCommon.hpp]: Fix copyright year

File: include/terabee/ITerarangerEvo40m.hpp [hunk 5]
```diff
@@ -0,0 +1,74 @@
+#ifndef I_TERARANGER_EVO_40M_HPP
+#define I_TERARANGER_EVO_40M_HPP
+
+/**
+ * @Author Pawel Ptasznik
+ *
+ * @Copyright Terabee 2020
+ *
+ */
```
Comment (pl-kabaradjian) [inline include/terabee/ITerarangerEvo40m.hpp:9]: Fix author and copyright year

File: include/terabee/ITerarangerEvo15m.hpp [hunk 6]
```diff
@@ -0,0 +1,74 @@
+#ifndef I_TERARANGER_EVO_15M_HPP
+#define I_TERARANGER_EVO_15M_HPP
+
+/**
+ * @Author Pawel Ptasznik
+ *
+ * @Copyright Terabee 2020
```
Comment (pl-kabaradjian) [inline include/terabee/ITerarangerEvo15m.hpp]: Fix author and copyright year

File: examples/ExampleReadEvo15m.cpp [hunk 7]
```diff
@@ -0,0 +1,47 @@
+/**
+ * @Author Pawel Ptasznik
+ *
+ * @Copyright Terabee 2020
+ *
+ */
```
Comment (pl-kabaradjian) [inline examples/ExampleReadEvo15m.cpp:6]: Fix author and copyright year

File: examples/ExampleReadEvo40m.cpp [hunk 8]
```diff
@@ -0,0 +1,47 @@
+/**
+ * @Author Pawel Ptasznik
+ *
+ * @Copyright Terabee 2020
+ *
+ */
```
Comment (pl-kabaradjian) [inline examples/ExampleReadEvo40m.cpp:6]: Fix author and copyright year

File: src/teraranger/TerarangerEvo15m.cpp [hunk 9]
```diff
@@ -0,0 +1,49 @@
+/**
+ * @Author Pawel Ptasznik
+ *
+ * @Copyright Terabee 2020
+ *
+ */
```
Comment (pl-kabaradjian) [inline src/teraranger/TerarangerEvo15m.cpp:6]: Fix author and copyright year

File: src/teraranger/TerarangerEvo40m.cpp [hunk 10]
```diff
@@ -0,0 +1,49 @@
+/**
+ * @Author Pawel Ptasznik
+ *
+ * @Copyright Terabee 2020
+ *
+ */
```
Comment (pl-kabaradjian) [inline src/teraranger/TerarangerEvo40m.cpp:6]: Fix author and copyright year

**Review by pl-kabaradjian — CHANGES_REQUESTED** [review]
As @pptaszni  pointed out, there are a few changes to address.

**Comment by mapi-ng** [comment]
Thanks for feedback, see the recent changes.

**Review by pl-kabaradjian — APPROVED** [review]

**Pull request merged**